src/pykeen/triples/__init__.py

```diff
@@ -56,12 +56,23 @@ class also provides rich factory methods that allow creating mappings from strin
 (:mod:`pykeen.triples.analysis`).
 """
 
-from .instances import BatchedSLCWAInstances, Instances, LCWAInstances, SubGraphSLCWAInstances
+from .instances import (
+    BaseBatchedSLCWAInstances,
```

Comment:
Do we want to expose this intermediate class as part of the public API?

Reply:
yeah, since I think this is the right level of abstraction to refer to with type annotations